### ISSUE TITLE
bug wrong global bin path

### ISSUE THREAD

**Issue by jas99** [yarnpkg/yarn#5328]
globals bin stoped working with node 9.5.0

> yarn global bin
`/usr/local/bin`

although dir seem to be outputted right.

> yarn global dir
`/Users/jas/.config/yarn/global`

yarn install via brew on high sierra 
> yarn versions
```
{ yarn: '1.3.2',
  http_parser: '2.7.0',
  node: '9.5.0',
  v8: '6.2.414.46-node.18',
  uv: '1.19.1',
  zlib: '1.2.11',
  ares: '1.13.0',
  modules: '59',
  nghttp2: '1.29.0',
  napi: '2',
  openssl: '1.0.2n',
  icu: '60.2',
  unicode: '10.0',
  cldr: '32.0.1',
  tz: '2017c' 
}
```

**Comment by rally25rs** [comment]
Those are the correct paths as far as I can tell.

```
~/Projects/yarn-test 🐒   yarn global bin
/usr/local/bin
~/Projects/yarn-test 🐒   yarn global dir
/Users/jvalore/.config/yarn/global
```

When Yarn installs a global, it makes symlinks in `/usr/local/bin` to the bin file in `/Users/{user}/.config/yarn/global`

```
~/Projects/yarn-test 🐒   ls -l `yarn global bin`/dodgem
lrwxr-xr-x  1 jvalore  admin  59 Dec  4 15:15 /usr/local/bin/dodgem -> /Users/jvalore/.config/yarn/global/node_modules/.bin/dodgem
```

Maybe an upgrade broke those links or something? Check the output of
```
ls -l `yarn global bin` | grep yarn

ls -l `yarn global dir`/node_modules/.bin`
```
and see if the links all line up. If you upgraded versions of yarn, I think there was a change at some point to how links were made, so that might have inadvertently broke something.

**Comment by jas99** [comment]
You are right all symlinks except the ones I upgraded with `yarn global upgrade-interactive` are missing. One more strange thing happened the upgraded packages were also added to my local package I was CD'ed into.

We have some inbuilt trick to repair alias?
